src/main/java/org/spongepowered/api/text/Texts.java

```diff
@@ -375,6 +506,19 @@ public static TextBuilder builder() {
     }
 
     /**
+     * Creates a new unformatted {@link TextBuilder.Placeholder} with the
+     * specified textArgument.
+     *
+     * @param textArgument The textArgument of the placeholder
+     * @return The created placeholder builder
+     * @see Text.Placeholder
+     * @see TextBuilder.Placeholder
+     */
+    public static TextBuilder.Placeholder placeholderBuilder(TextArgument textArgument) {
```

Comment:
The String version already uses this name, and I cannot change it since it takes the same method params.